Algorithms/FedBe.py

```diff
@@ -0,0 +1,13 @@
+from Servers.FedBeServer import FedBeServer
+from Clients.FedAvgClient import FedAvgClient
+from Algorithms.ABCAlgorithm import ABCAlgorithm
+
+class FedBe(ABCAlgorithm):
+    def __init__(self,dataloader,Model,loc_data,callbacks=None,n_clients=5,M=10,
```

Comment:
Arguably we only need default values either here or in the server. Keeping the default values at the server seem reasonable and ot make anything more than a default FedBE algorithm you'd access  the server specifically through the algorithm object.

Reply:
Removed in latest commit.